example/mm_jobman.sh

```diff
@@ -208,20 +212,30 @@ submit_each_line_with_mmfloat() {
         if [ "$dryrun" = true ]; then
             full_cmd+="#-------------\n"
         fi
-        cmd="$download_cmd$line$upload_cmd" 
-        full_cmd+="mmfloat submit -i '$image' -j <(echo -e '''$cmd''') -c '$c_value' -m '$m_value' $dataVolume_params\n"
- 
-    done < "$script_file"
 
-    # Remove the last '\n'
-    full_cmd=${full_cmd%\\n}
+        image_run=$(echo $line | awk '{print $3}')
+	subline+="mkdir -p /home/jovyan/TEMP/output && "
```

Comment:
mkdir is taken care of for --download and --cwd (command to be added) so here there should not be a hard-coded mkdir. Please remove.